### PR TITLE
Added the possibility of choosing your class in the prep time

### PR DESCRIPTION
**Pull request by lebroomer**

### CODE REVIEW

**Comment by lebroomer** [comment]
- activatable with the ConVar "tttc_choose_class 1"

- Add a custom icon to your class by adding a member named "icon" to the "classData"-structure with the path to an icon (e.g. "/vgui/icon.png").

File: lua/classes/client/cl_hud.lua [hunk 1]
```diff
@@ -40,182 +92,360 @@ local margin = 10
...
+		DrawIconFramed(cd.icon, x + tw / 2 - 64, y + 110, 128, 128, x, x + tw)
+		
+		foundWeapon = true
+	end
+	
+	-- weapons
+	if bWeapons then
+		for _, cls in ipairs(cd.weapons) do
+			local tmp = weapons.Get(cls)
+			if tmp and tmp.Icon and not foundWeapon then -- TODO "icon" not "Icon"? ll 137
```
Comment (Alf21) [inline lua/classes/client/cl_hud.lua]: @lebroomer should it be "icon" or "Icon" ?

Comment (lebroomer) [reply lua/classes/client/cl_hud.lua]: No "Icon" is correct. It is the standard SWEP member for TTT weapons (it defines the icon shown in the traitor shop)

Comment (Alf21) [reply lua/classes/client/cl_hud.lua]: alright and the var "icon" in line 137? Your own created / custom var?

Comment (lebroomer) [reply lua/classes/client/cl_hud.lua]: correct, thats the member of the class data, where you can put the path to your custom class icon.

**Pull request merged**